introduction.md

```diff
@@ -3,3 +3,11 @@
 My name is Victor Manuel Espiridion and I love to work with Git.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?